include/openthread/srp_client.h

```diff
@@ -670,6 +670,30 @@ void otSrpClientSetServiceKeyRecordEnabled(otInstance *aInstance, bool aEnabled)
  */
 bool otSrpClientIsServiceKeyRecordEnabled(otInstance *aInstance);
 
+/**
+ * Enables/Disables use of SRP coder to send coded SRP update message.
```

Comment:
Here it's unclear what "enabled" means. If it is set to enabled for example, but all of the SRP Registries found in the mesh are "version 0"  (known to not support the SRP-code format), then is coded SRP Update used, or not used?
Can it be enabled and not used in this case? Or is that "disabled" then?

Reply:
@EskoDijk , please see this [commet](https://github.com/openthread/openthread/pull/10606#issuecomment-2767652186) below related to this.